### DISCUSSION TITLE
I am Interested in making Outreachy contribution for May 2024 cohort

### DISCUSSION THREAD

**Discussion by OlawumiSalaam** [ModECI/MDF#505] [General]
Hello. My name is Olawumi Salaam. The MDF project piqued my interest due to my experience with building machine learning models and model deployment on streamlit. I am looking forward to meaningful learnings and contribution as I await further instructions.

**Comment by Iqra350** [comment]
@OlawumiSalaam I have not get back from the admin i am not sure what would be the next process to do. In case you find any communication channel please let me know as well

**Comment by ai-naymul** [comment]
@pgleeson @sanjayankur31 

HI there,
Is there any specific communication channel for outreachy internship period or should I create a thread in GitHub discussion regarding that?

**Reply by chickenleaf** [reply]
I'm wondering if i should send one of our mentors a mail regarding this.

**Reply by ai-naymul** [reply]
> I'm wondering if i should send one of our mentors a mail regarding this. 

Lets see I mention them in my message if they reply on this then there is no need to send any mail if not we will plan to send mail to mentors :)

**Comment by Iqra350** [comment]
I think its good idea. Please contact any of them and see what public channel they suggest for us for discussion

**Reply by ai-naymul** [reply]
Lets wait for some time if not we should consider contacting them👍

**Reply by Iqra350** [reply]
Any one of us can contact.

**Comment by pgleeson** [comment]
Hi all, thanks for your interest in the project!
I'll be posting more information in the Discussions shortly.

**Reply by rasmodev** [reply]
@pgleeson Thank you for the update.

**Reply by OlawumiSalaam** [reply]
@ Thank you for your response @pgleeson

**Reply by Olawumi-11** [reply]
Ok

**Comment by pgleeson** [comment]
Hi all! See https://github.com/ModECI/MDF/discussions/509 for some more details on the project.
